vNext/extensions/applicationinsights-react-js/README.md

```diff
@@ -3,20 +3,86 @@
 [![Build Status](https://travis-ci.org/Microsoft/ApplicationInsights-JS.svg?branch=master)](https://travis-ci.org/Microsoft/ApplicationInsights-JS)
 [![npm version](https://badge.fury.io/js/%40microsoft%2Fapplicationinsights-react-js.svg)](https://badge.fury.io/js/%40microsoft%2Fapplicationinsights-react-js)
 
+React Plugin for the Application Insights Javascript SDK, enables the following:
+
+- Tracking of router changes
+- React components usage statistics
+
 React Plugin for the Application Insights Javascript SDK
 
-## Build:
-```
-npm install -g grunt-cli
-npm install
-npm run build --silent
-```  
+## Getting Started
+
+Install npm package:
 
-## Run unit tests:
+```bash
+npm install --save @microsoft/applicationinsights-react-js
 ```
-npm run test
+
+## Basic Usage
+
+### npm Setup 
+```js
+import React from 'react';
+import { ApplicationInsights } from '@microsoft/applicationinsights-web';
+import { ReactPlugin, withAITracking } from '@microsoft/applicationinsights-react-js';
+import { createBrowserHistory } from "history";
+
+const browserHistory = createBrowserHistory({ basename: '' });
+var reactPlugin = new ReactNativePlugin();
+var appInsights = new ApplicationInsights({
+    config: {
+        instrumentationKey: 'YOUR_INSTRUMENTATION_KEY_GOES_HERE'
+    },
+    extensions: [reactPlugin],
+    extensionConfig: {
+      [reactPlugin.identifier]: { history: browserHistory }
+    }
+});
+appInsights.loadAppInsights();
+
+// To instrument various React components usage tracking, apply the `withAITracking` higher-order
+// component function.
+
+class MyComponent extends React.Component {
+    ...
+}
+
+export default withAITracking(appInsights, MyComponent);
 ```
 
+## Configuration
+
+| Name | Default | Description |
+|------|---------|-------------|
+| history | null | React router history for more information see the [documentation][react-router] of the `react-router` package. |
+
```

Comment:
Does this configuration support "debug" mode? It's good to see the component activity when debug mode is set.

Reply:
We don't support debug to console anywhere in the SDK, I know is convenient during testing or development but the same can be achieved with any browser debugger so we are not planning to include it for now.